Makefile

```diff
@@ -89,7 +95,7 @@ install:
 	mkdir -p ${DESTDIR}/bin ${HANDLERSDIR}
 	cp ${BIN_NAME} ${DESTDIR}/bin/${BIN_NAME}
 	cp handlers.yml ${HANDLERSDIR}
-	cp -f handlers/* ${HANDLERSDIR} || :
+	cp -rf handlers/* ${HANDLERSDIR} || :
```

Comment:
I think you're doing the recursive copy, because you want to install also the `gradle_handler/node_modules/`, right ?
The problem is that it'll install everything from the `*_handler/` directories (for example the `rust_handler/` contains 30MB of rust libraries after you build rust handler).

I'd say:

- add a Makefile to `gradle_handler/` which would do:
  - `npm install .`
  - `cp gradle ../`
- run the gradle_handler/make in `handlers:` section of **this** Makefile
- in **this** `install:` section just add explicit `cp -rf handlers/handler_gradle/node_modules/ ${HANDLERSDIR} || :`

Reply:
Something like I do in #35